docker-compose/docker-compose.yml

```diff
@@ -186,7 +186,7 @@ services:
 
   neon-test-extensions:
     profiles: ["test-extensions"]
-    image: ${REPOSITORY:-neondatabase}/neon-test-extensions-v${PG_TEST_VERSION:-16}:${TAG:-latest}
+    image: ${REPOSITORY:-neondatabase}/neon-test-extensions-v${PG_TEST_VERSION:-16}:${TEST_EXTENSIONS_TAG:-${TAG:-latest}}
```

Comment:
@a-masterov right now, I'm setting this to `latest`, but it feels like this should point to either the currently being built thing, or if that's not available because we're doing a storage/proxy release, the most recent compute release. Pointing it to the most recent compute release didn't work because of https://github.com/neondatabase/neon/pull/10943, but in the long run, pointing it to the most recent compute release would be the correct behaviour, right?